docs/geedocs/5.3.0/answer/7160007.html

```diff
@@ -111,6 +117,11 @@ <h5>
               Help has been added through the <code>--help</code> option, or if any invalid parameter is given
             </td>
           </tr>
+          <tr>
+            <td>1194</td>
+            <td>Sockets build up during large project builds</td>
+            <td>The Fusion UI and <code>getop</code> will now timeout when attempting to communicate with System Manager in order to avoid keeping sockets open indefinitely. The timeout is configurable.</td>
```

Comment:
Minor: technically SystemManager is the side which will timeout so it doesn't build up a backlog of requests while another task is holding the lock.
There's also a new configurable parameter for `misc.xml` (`MutexTimedWaitSec`).  I'm not sure if that's something that goes in the release notes or not.

Reply:
Agreed that release notes may not be the best place, but it does seem like it should go somewhere. Probably in 176738.html (Configure Fusion performance). I'll put a note in there and update the wording here to be more accurate.